README.md

```diff
@@ -29,4 +29,4 @@
 
 ## Our really nice logo! 🚀
 
-![ape](https://user-images.githubusercontent.com/92869397/166645877-e8570f35-82fd-41cb-a702-3b5d1a3068a0.JPG)
\ No newline at end of file
+![ape](https://user-images.githubusercontent.com/92869397/166645877-e8570f35-82fd-41cb-a702-3b5d1a3068a0.JPG)
```

Comment:
what did actually change in this line?

Reply:
@lukas-rotsching I guess it was just because of the auto-formater. So no actual change